### PR TITLE
Install and update dotenv, add proxy

### PR DESCRIPTION
**Pull request by SGierszewski**

### CODE REVIEW

**Review by dhnazarenko — APPROVED** [review]
LGTM 🚀

**Review by JGieseWeb — APPROVED** [review]
👏 good work

File: vite.config.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
 import { defineConfig } from "vite";
 import reactRefresh from "@vitejs/plugin-react-refresh";
+import dotenv from "dotenv";
+
+dotenv.config();
```
Comment (fgottesmann) [inline vite.config.js]:
```suggestion
import dotenv from "dotenv";

dotenv.config();
import { defineConfig } from "vite";
import reactRefresh from "@vitejs/plugin-react-refresh";

```

**Review by fgottesmann — APPROVED** [review]

**Review by MaxMuecket — APPROVED** [review]
lgtm

**Pull request merged**